README.md

```diff
@@ -1 +1,97 @@
+         1. Announced at one (1) regular meeting, at least one (1) week, in advance of the election.
+         2. An email be sent to all members by the Steering Committee upon receipt of the petition.
+8. Along with the recall process which may be initiated by members, members of the Steering Committee may be removed as a result of a grievance when they have been found to be negligent or mishandle their power and responsibilities.
+   a. Unilateral decision making, abuse of power, domineering discussion/coercing decisions in conversations, breaches of confidentiality or discretion and/or any other actions which constitute the need for an investigation into undemocratic behavior fall under the category of malicious mishandling of vested power.
+   b. The accused party(ies) of the Steering Committee shall not be present in any meetings discussing their supposed transgression or the resolutions to follow any investigations they are involved in.
+   c. Membership shall be notified in all cases of a change in the composition of the Steering Committee, at a minimum doing the following:
+      1. Announced at the next regular meeting following the change of composition.
+      2. An email be sent to all members by the Steering Committee upon change of composition.
+9. The duties of the Co-Chairs shall be:
+   a. To facilitate meetings.
+   b. To write updates to send via email to everyone on the organization's email list at least once per month.
+10. The duties of the Secretary-Treasurer shall be:
+   a. To take minutes at general meetings.
+   b. To send meeting minutes to everyone on the organization's email list within ninety-six (96) hours of a meeting's end.
+   c. To maintain a list of active members and build an email list.
+   d. To send email updates written by the Co-Chairs to membership.
+   e. To manage any funds of the organization.
+   f. To maintain an active log of the Organization's income and expenditures.
+   g. To serve as the lead organizer for all fundraisers.
+
+## ARTICLE V: FUNDING
+
+1. The organization shall be funded through the following means: fiscal sponsorship, fundraisers, contributions from any organization or individual that supports the Organization's mission, fundraisers, and any approved financial assistance granted by the Organization's steering committee.
+2. All money transactions must be approved by the Secretary-Treasurer.
+3. All money transactions over $100 must be approved by a majority vote of the membership at a regular meeting.
+4. The Steering Committee shall maintain a record of all financial transactions and accounts. These records shall be made available at any time by the request of any active member of the Organization and the general public.
+
+## ARTICLE VI: MEETINGS
+
+1. Regular meetings of the organization shall be held on the second Saturday of each calendar month.
+   a. Meeting times may be changed by a one-thirds (⅔) vote of membership.
```

Comment:
```suggestion
   a. Meeting times may be changed by a two-thirds (⅔) vote of membership.
```

Reply:
It was initially two-thirds. The issue is we have very high membership numbers of people who aren't super involved, so doing things that require 2/3 is extremely difficult to get. 

although thinking abt it now, only people who vote will count so does it matter?

Reply:
Yeah, I think this is best left to a discussion at our meeting.